org.pgadmin.pgadmin4.yml

```diff
@@ -100,12 +100,14 @@ modules:
       - tar -xf data.tar.zst
       - rm -rf usr/pgadmin4/venv
       - cp -r usr/* /app/
-      - sed -i "233 i DATA_DIR = os.environ['XDG_CONFIG_HOME']" /app/pgadmin4/web/config.py
+      - cat config_distro.py usr/pgadmin4/web/config_distro.py >> ${FLATPAK_DEST}/pgadmin4/web/config_distro.py
```

Comment:
Doesn't it duplicate existing config_distro.py content? Two files in cat + append to existing

Reply:
It's duplicated:
```
$ cat /var/lib/flatpak/app/org.pgadmin.pgadmin4/current/active/files/pgadmin4/web/config_distro.py 
HELP_PATH = '../../../share/docs/en_US/html/'
DEFAULT_BINARY_PATHS = {
    'pg':   '/usr/bin',
    'ppas': ''
}
import os

DATA_DIR = os.environ["XDG_CONFIG_HOME"]
HELP_PATH = '../../../share/docs/en_US/html/'
DEFAULT_BINARY_PATHS = {
    'pg':   '/usr/bin',
    'ppas': ''
}
```

Reply:
Should be better